sdk/textanalytics/Azure.AI.TextAnalytics/src/AnalyzeHealthResultCollection.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using Azure.AI.TextAnalytics.Models;
+
+namespace Azure.AI.TextAnalytics
+{
+    /// <summary>
+    /// Collection of <see cref="DetectLanguageResult"/> objects corresponding
+    /// to a batch of documents, and information about the batch operation.
+    /// </summary>
+    public class AnalyzeHealthResultCollection : ReadOnlyCollection<HealthcareResult>
+    {
+        /// <summary>
+        /// </summary>
+        /// <param name="list"></param>
+        /// <param name="statistics"></param>
+        /// <param name="modelVersion"></param>
+        internal AnalyzeHealthResultCollection(IList<HealthcareResult> list, TextDocumentBatchStatistics statistics, string modelVersion) : base(list)
+        {
+            Documents = list;
+            Statistics = statistics;
+            ModelVersion = modelVersion;
+        }
+
+        /// <summary>
+        /// Gets statistics about the documents batch and how it was processed
+        /// by the service.  This property will have a value when IncludeStatistics
+        /// is set to true in the client call.
+        /// </summary>
+        public IList<HealthcareResult> Documents { get; }
```

Comment:
This shouldn't be a property as the list should be passed to the ReadOnlyCollection